content/en/news/releases/1.19.x/announcing-1.19/change-notes/index.md

```diff
@@ -0,0 +1,302 @@
+---
+title: Istio 1.19.0 Change Notes
+linktitle: 1.19.0
+subtitle: Minor Release
+description: Istio 1.19.0 change notes.
+publishdate: 2023-09-01
+release: 1.19.0
+weight: 20
+---
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/docs/releases/feature-stages/#feature-phase-definitions). Please consider upgrading your environment to remove the deprecated functionality.
+
+- There are no new deprecations in Istio 1.19.0
+
+## Traffic Management
+
+- **Improved** JWT claim based routing, now support using `[]` as a separator for nested claim names.
+  ([Issue #44228](https://github.com/istio/istio/issues/44228))
+
+- **Improved** performance of sidecar injection, in particular with pods with a large number of environment variables.
+
+- **Updated** When using a ServiceEntry with DNS resolution, DNS for multi-network gateways
+will be resolved at the proxy instead of in the control plane.
+
+- **Added** support for `traffic.sidecar.istio.io/excludeInterfaces` annotation in proxy.
+  ([Issue #41271](https://github.com/istio/istio/issues/41271))
+
+- **Added** `WorkloadEntry` resources on different networks do not require an address to be specified.
+  ([Issue #45150](https://github.com/istio/istio/issues/45150))
+
+- **Added** initial ambient support for `WorkloadEntry`
```

Comment:
Should we have a dedicated section for ambient?

Reply:
I think that is better in the announcements section